### PR TITLE
Pr/misc

### PR DESCRIPTION
**Pull request by pavanbalaji**

### CODE REVIEW

File: src/mpi/comm/comm_split_type.c [hunk 1]
```diff
@@ -27,21 +27,43 @@ int MPI_Comm_split_type(MPI_Comm comm, int split_type, int key, MPI_Info info, M
 #define MPI_Comm_split_type PMPI_Comm_split_type
 
 #undef FUNCNAME
-#define FUNCNAME MPIR_Comm_split_type_impl
+#define FUNCNAME MPIR_Comm_split_type
 #undef FCNAME
 #define FCNAME MPL_QUOTE(FUNCNAME)
-int MPIR_Comm_split_type_impl(MPIR_Comm * comm_ptr, int split_type, int key,
-                              MPIR_Info * info_ptr, MPIR_Comm ** newcomm_ptr)
+int MPIR_Comm_split_type_shmem_node(MPIR_Comm * comm_ptr, int key, MPIR_Comm ** newcomm_ptr)
```
Comment (hajimefu) [inline src/mpi/comm/comm_split_type.c]: Why are `FUNCNAME` and the actual function name different?

Comment (pavanbalaji) [reply src/mpi/comm/comm_split_type.c]: Fixed.

**Comment by pavanbalaji** [comment]
test:jenkins/ch3/tcp

**Comment by pavanbalaji** [comment]
test:jenkins/ch3/tcp

**Comment by yfguo** [comment]
The PR look fine. I noticed 387b06b is no longer needed because the unused local variables are removed in the latest master (probably because of a recent merge).

**Comment by pavanbalaji** [comment]
test:jenkins/ch3/tcp

**Pull request merged**